sdks/python/apache_beam/ml/inference/vertex_ai_inference_it_test.py

```diff
@@ -45,6 +47,10 @@
 
 
 class VertexAIInference(unittest.TestCase):
+  @unittest.skipIf(
+      tf is None, 'Missing dependencies. '
+      'Test depends on tensorflow')
+  @pytest.mark.uses_tf
```

Comment:
is this necessary?

Reply:
it's ok too keep. looks like we alredy defined this marker and use it in other tests.

Reply:
Perhaps not but considering the postcommits went red without a clear reason I'd rather cover all of our bases here.  would have thought that missing TF would have gotten caught in the try-except-skip import block getting the pipeline code, but it didn't.

Reply:
actually, i take it back. I think this decorator is serving a different puprose, see 

https://github.com/apache/beam/blob/3915c85b84207f5f2c1e9dc4dd441769f3c586a5/sdks/python/tox.ini#L398

Reply:
adding this annotation would make tox pick up this test some other unit test suites, and then skip b/c vertex ai deps are not installed.